app/Controllers/WordController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+use App\Entities\Response;
+use App\Enumerators\HttpHeaderStatus;
+use App\Exception\NotFoundException;
+use App\Exception\UnprocessableEntityException;
+use App\Repositories\WordRepository;
+
+readonly class WordController implements ControllerInterface
+{
+    public function __construct(
+        private WordRepository $wordRepository,
+    ) {
+    }
+
+    public function getAll(): Response
+    {
+        $result = $this->wordRepository->getAllWords();
+
+        return new Response(HttpHeaderStatus::OK->value, json_encode($result));
+    }
+
+    public function getById(int $id): Response
+    {
+        $wordEntity = $this->wordRepository->getWordById($id);
+        if($wordEntity === null) {
```

Comment:
Let your code breath, ad space before if to visually separate different segments of logic